### PR TITLE
Included new CVE's

### PR DESCRIPTION
**Pull request by alagujeeva**
## Describe the Change

This PR .....

## Review Changes

💻 [Preview URL](https://deploy-preview-1668--docs-spectrocloud.netlify.app/security/security-bulletins/cve-reports)

🎫 [Jira Ticket]()

### CODE REVIEW

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 1]
```diff
@@ -441,4 +441,151 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
 
 #### References
 
-- [CVE-2022-1292](https://nvd.nist.gov/vuln/detail/CVE-2022-1292)
\ No newline at end of file
+- [CVE-2022-1292](https://nvd.nist.gov/vuln/detail/CVE-2022-1292)
+
+## September 25, 2023 - CVE-2023-42810 systeminformation Library for NodeJS have a SSID Command Injection Vulnerability - 9.8 CVSS
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply docs/docs-content/security/security-bulletins/cve-reports.md:446]: Yes

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 2]
```diff
@@ -441,4 +441,151 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
...
+
+systeminformation is a System Information Library for Node.JS. Versions 5.0.0 through 5.21.6 have a SSID Command Injection Vulnerability. The problem was fixed with a parameter check in version 5.21.7. As a workaround, check or sanitize parameter strings that are passed to `wifiConnections()`, `wifiNetworks()` (string only).
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply docs/docs-content/security/security-bulletins/cve-reports.md:448]: Yes

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 3]
```diff
@@ -441,4 +441,151 @@ Update cluster profiles using the vSphere CSI pack to version v2.6 or greater. A
...
+
+
+<br />
+
+#### Impact
+
+To perform a potential command injection, need to craft detected SSIDs in wifiConnections(), wifiNetworks(), which is not used by us.
```
[the remark this answers is not in the document]
Comment (alagujeeva) [reply docs/docs-content/security/security-bulletins/cve-reports.md:455]: Yes

File: docs/docs-content/security/security-bulletins/cve-index.md
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-index.md:1]: @alagujeeva I left some questions in the security channel.

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 4]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
 -->
 
+## October 11, 2023 - CVE-2023-44487 HTTP/2 Denial of Service - 7.5 CVSS
+
+The HTTP/2 protocol may be used to create a denial of service and cause a server to exhaust all of its allocated resources. A malicious HTTP/2 client which rapidly creates requests and
+immediately resets them can cause excessive server resource consumption.
+
+
+
+#### Impact
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:44]: What versions of Palette are impacted?

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 5]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
+
+Denial of Service
+
+
+
+#### Resolution
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:50]: See Slack thread for the question I asked about this part.

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 6]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
+
+Palette Go based binaries are compiled with go1.21.3 which has the fix.
+
+
+
+#### Patches
+
+Not Applicable
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:58]: See Slack thread for the question I asked about this part.

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 7]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
+
+- [CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)
+
+## October 17, 2023 - CVE-2023-4911 Buffer Overflow in Dynamic Loader - 7.8 CVSS
+
+A buffer overflow was discovered in the GNU C Library’s dynamic loader `ld.so` while processing the `GLIBC_TUNABLES` environment variable. This issue could allow a local attacker to use maliciously crafted `GLIBC_TUNABLES` environment variables when launching binaries with `SUID`` permission to execute code with elevated privileges.
+
+
+
+#### Impact
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:71]: What versions of Palette are impacted?

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 8]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
+
+The attacker needs to have access to an Ubuntu instance locally to launch an attack.
+
+
+
+#### Patches
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:77]: What versions does the user need to upgrade to?

File: docs/docs-content/security/security-bulletins/cve-reports.md [hunk 9]
```diff
@@ -32,6 +34,149 @@ _Are there any links users can visit to find out more?_
...
+
+
+## October 6, 2023 - CVE-2023-32002 NodeJS Modules Policy Bypass - 9.8 CVSS
+
+The use of the Module library's `Module._load()` function can be used to bypass the defined policy mechanism and require external modules not defined in the  **policy.json** file for a given module. This vulnerability affects all users using the experimental policy mechanism in all active release lines: 16.x, 18.x and, 20.x. Note that at the time this CVE was issued, the policy is an experimental feature of Node.js.
+
+
+
+#### Impact
+
```
Comment (karl-cardenas-coding) [inline docs/docs-content/security/security-bulletins/cve-reports.md:101]: What versions of Palette are affected? If none, then let's state that.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
@alagujeeva Some of the CVEs are not answering the questions we have specified in the template at the top of the file. Please take a look at the template so that you can get familiar with the questions that need to be answered.

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**